x/tally/keeper/endblock.go

```diff
@@ -94,11 +94,11 @@ func (k Keeper) ProcessTallies(ctx sdk.Context, coreContract sdk.AccAddress) err
 		}
 
 		switch {
-		case len(req.Commits) < int(req.ReplicationFactor):
+		case len(req.Commits) == 0 || len(req.Commits) < int(req.ReplicationFactor):
 			dataResults[i].Result = []byte(fmt.Sprintf("need %d commits; received %d", req.ReplicationFactor, len(req.Commits)))
 			dataResults[i].ExitCode = batchingtypes.TallyExitCodeNotEnoughCommits
 			k.Logger(ctx).Info("data request's number of commits did not meet replication factor", "request_id", req.ID)
-		case len(req.Reveals) < int(req.ReplicationFactor):
+		case len(req.Reveals) == 0 || len(req.Reveals) < int(req.ReplicationFactor):
```

Comment:
I think we should handle this case differently, as receiving 9/10 reveals can still potentially lead to a result where 7 answers are the same (or considered similar enough when using deviation). I guess that does mean we have to pass the reveals and RF separately, or we use a `nil` value to signal a missing reveal.

Reply:
Good idea and I think it makes sense to address this in a separate PR together with issue #453 ?